sys/page_revisions.js

```diff
@@ -373,16 +536,19 @@ PRS.prototype.getTitleRevision = function(hyper, req) {
                 // In case 404 is returned by MW api, the page is deleted
                 .then(function(result) {
                     result = result.body.items[0];
-                    result.tid = uuid.now().toString();
-                    result.restrictions = result.restrictions || [];
-                    result.restrictions.push('page_deleted');
-                    return hyper.put({
-                        uri: self.tableURI(rp.domain),
-                        body: {
-                            table: self.tableName,
-                            attributes: result
-                        }
-                    }).throw(e);
+                    result.tid = TimeUuid.now().toString();
+                    result.page_deleted = result.rev;
```

Comment:
When we switch completely to the new access checks model we'd be able to remove this. TODO might be nice.